src/main/java/com/box/sdk/BoxWebHookSignatureVerifier.java

```diff
@@ -0,0 +1,253 @@
+package com.box.sdk;
```

Comment:
There is no additional third party library. We use "javax.crypto". Which is direct part of Java SDK. Potential issue can be with non Oracle JDK, e.g.: OpenJDK, but it uses only simple "HmacSHA256", which I think is supported by OpenJDK since 2003. I am not sure about other Java SDK implementations.